### DISCUSSION TITLE
Cannot disable full text search after meilisearch free trial expired

### DISCUSSION THREAD

**Discussion by SE-NGO-Team** [AntlerVC/firetable#1440] [Support / Q&A] [answered]
<img width="602" alt="image" src="https://github.com/rowyio/rowy/assets/145421295/d3c4573c-dee4-4614-8844-1543d45b7923">
I cannot change to my another instance and it keep stuck in the page for Disable table search

**Comment by Chadha93** [comment, marked as answer]
HI @SE-NGO-Team,

If you are unable to disable full text search on MeiliSearch after your free trial has expired, it could be due to limitations imposed on your account. Once your trial expires, certain features may no longer be accessible.

To resolve this issue, you will need to upgrade your MeiliSearch account to a paid plan.
